samples/nrf9160/aws_fota/src/main.c

```diff
@@ -438,32 +441,43 @@ static int fds_init(struct mqtt_client *c)
  *
  * Blocks until link is successfully established.
```

Comment:
The doxygen for this function is no longer valid

Reply:
Fixed.